aws_lambda_powertools/logging/correlation_paths.py

```diff
@@ -6,4 +6,5 @@
 APPSYNC_RESOLVER = 'request.headers."x-amzn-trace-id"'
 APPLICATION_LOAD_BALANCER = 'headers."x-amzn-trace-id"'
 EVENT_BRIDGE = "id"
+LAMBDA_FUNCTION_URL = "requestContext.requestId"
```

Comment:
```suggestion
LAMBDA_FUNCTION_URL = API_GATEWAY_REST
```

As they follow the same payload specification